tools/sof-dump-status.py

```diff
@@ -282,29 +291,14 @@ def export_cardinfo_pcm(card_info):
 
         def _getStr(tmp_dict, key):
             return '%s=%s'%(key, tmp_dict.get(key))
-
-        def _covertType(type_lst):
-            ret=0
-            for type_str in type_lst:
-                if type_str.split()[0].lower() == 'playback':   ret += 1
-                elif type_str.split()[0].lower() == 'capture':  ret += 2
-                # here maybe have some problem, so don't need to check it
-                else:   return ""
-            if ret == 1:    return 'playback'
-            elif ret == 2:  return 'capture'
-            else:           return 'both'
-
         for idx in range(0, length):
             pcm = pcm_lst[idx]
-            print('%s_LST[%d]="%s;%s;%s=%s"' % (keyword, idx, _getStr(pcm, 'id'), _getStr(pcm, 'pcm'), 'type', ', '.join(pcm.get('type'))))
-            # date format for match tplg format
-            pcm['pcm'] = pcm['pcm'].replace('(*)', '').strip()
-            pcm['type'] = _covertType(pcm['type'])
-            # store pcm to each list
-            print('unset %s_%s' % (keyword, pcm['id']))
-            print('declare -Ag %s_%s' % (keyword, pcm['id']))
+            print('%s_LST[%d]="%s;%s;%s"' % (keyword, idx, _getStr(pcm, 'id'), _getStr(pcm, 'pcm'), _getStr(pcm, 'type')))
+            # date format for match tplg format store pcm to each list
```

Comment:
```suggestion
            # Use date format for matched tplgs when storing pcms to each list
```
If that doesn't make sense, feel free to ignore this suggestion --- I struggled a bit trying to understand what you were trying to say.